### PR TITLE
chore: deprecate solar theme

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

Solar theme is considered legacy and no longer valid for Workspace.

Fixes # (issue)
DME-18976

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by Theeraphat-Sorasetsakul — DISMISSED** [review]
LGTM.

File: README.md [hunk 1]
```diff
@@ -21,13 +21,15 @@ Getting started and usage guide are available from this [documentation](https://
...
-| [@refinitiv-ui/solar-theme](https://github.com/Refinitiv/refinitiv-ui/tree/v7/packages/solar-theme)         | [![npm version](https://badgen.net/npm/v/@refinitiv-ui/solar-theme)](https://www.npmjs.com/package/@refinitiv-ui/solar-theme)         | [![change log](https://badgen.net/badge/icon/changelog/grey?icon=git&label)](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/solar-theme/CHANGELOG.md)     |
+| [@refinitiv-ui/solar-theme](https://github.com/Refinitiv/refinitiv-ui/tree/v7/packages/solar-theme)⚠️       | [![npm version](https://badgen.net/npm/v/@refinitiv-ui/solar-theme)](https://www.npmjs.com/package/@refinitiv-ui/solar-theme)         | [![change log](https://badgen.net/badge/icon/changelog/grey?icon=git&label)](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/solar-theme/CHANGELOG.md)     |
```
Comment (wattachai-lseg) [inline README.md]: Let's add one space.
```suggestion
| [@refinitiv-ui/solar-theme](https://github.com/Refinitiv/refinitiv-ui/tree/v7/packages/solar-theme) ⚠️       | [![npm version](https://badgen.net/npm/v/@refinitiv-ui/solar-theme)](https://www.npmjs.com/package/@refinitiv-ui/solar-theme)         | [![change log](https://badgen.net/badge/icon/changelog/grey?icon=git&label)](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/solar-theme/CHANGELOG.md)     |
```

File: packages/solar-theme/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Solar Theme
+# Solar Theme is deprecated.
 
 This is a legacy theme. It's only to use for Eikon products and services and shall not use within LSEG Workspace.
```
Comment (wattachai-lseg) [inline packages/solar-theme/README.md]: A sentence as a heading doesn't look quite right.
```suggestion
# Solar Theme

This is a **deprecated** legacy theme. It's only to use for Eikon products and services and shall not use within LSEG Workspace.
```

File: lerna.json [hunk 3]
```diff
@@ -8,6 +8,12 @@
       "registry": "https://registry.npmjs.org"
     }
   },
-  "ignoreChanges": ["**/__test__/**", "**/__demo__/**", "**/__snapshots__/**", ".vscode/**"],
+  "ignoreChanges": [
```
Comment (wattachai-lseg) [inline lerna.json]: Did anyone suggest you to update `ignoreChanges`? I don't think it's a good idea as [lerna docs](https://github.com/lerna/lerna/tree/main/libs/commands/version#--ignore-changes) say that there are some cases that a package will always be published, regardless of this option.

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]

File: lerna.json (hunk 3 above)
Comment (Theeraphat-Sorasetsakul) [reply lerna.json]: So that we will let Solar theme update when Elemental theme is changed, isn't it?

Comment (bualoy-napat) [reply lerna.json]: You're right. I will look for proper way to ignore the change in package.

Comment (wattachai-lseg) [reply lerna.json]: The goal is to clearly communicate that the package is deprecated. Releasing new versions might contradict with the deprecation but I don't think it's a big deal.

In short, I believe there is no need to stop publishing new versions. Trying to do so create an unnecessary risk - its behavior becomes uncertain.

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**